app/src/main/java/org/mozilla/focus/home/HomeFragment.java

```diff
@@ -494,12 +494,18 @@ public void onSwipeDown() {
 
         @Override
         public void onLongPress() {
-            doWithActivity(getActivity(), ThemeManager::resetDefaultTheme);
+            doWithActivity(getActivity(), (themeManager)->{
```

Comment:
[ant:checkstyle] [ERROR] /bitrise/src/app/src/main/java/org/mozilla/focus/home/HomeFragment.java:497:59: '->' is not followed by whitespace. [WhitespaceAround]
[ant:checkstyle] [ERROR] /bitrise/src/app/src/main/java/org/mozilla/focus/home/HomeFragment.java:497:59: '{' is not preceded with whitespace. [WhitespaceAround]
 FAILED